frontends/precinct-scanner/src/screens/scan_warning_screen.tsx

```diff
@@ -197,20 +222,13 @@ function UndervoteWarningScreen({
                   </span>
                 )}
               </p>
-              <Text italic>
+              <Text italic small>
```

Comment:
why are we also making the text smaller?

Reply:
It better matches the hierarchy of text on the screen before opening the modal